#263 add touch start event to close widget on touch devices

zolotyh opened the pull request.

Commit:
fix iphone

Commit:
remove comma

alexandernst commented:
@zolotyh I forked this project and applied your patch here https://github.com/alexandernst/angular-multi-select/commit/e048a14eaddaabdae84a6c95b6298e268822e8a9

You're welcome to send more patches.

nimrin committed:
Removed handlers for touch enabled devices that cause page freezing

these handlers cause page freezing on Apple devices if there are many multi-select components on one page; $scope.scrolled does not seem necessary.

zolotyh committed:
Merge pull request #1 from nimrin/master

Removed handlers for touch enabled devices that cause page freezing

nimrin committed:
Removed unbinding of nonexistent methods

Removed unbinding of nonexistent methods which invoke console errors

zolotyh committed:
Merge pull request #2 from nimrin/master

Removed unbinding of nonexistent methods

nimrin committed:
Prevent onSearchChange callback call

Prevent onSearchChange callback call if inputModel was changed programmatically

zolotyh committed:
Merge pull request #3 from nimrin/master

Prevent onSearchChange callback call

nimrin committed:
added disabling button

nimrin committed:
added class for disabled button

zolotyh committed:
Merge pull request #4 from nimrin/master

added disabled button

zolotyh committed:
fix bug with doule event with angular-touch

zolotyh committed:
fix typo

zolotyh committed:
fix typo

zolotyh committed:
fix bug with focus

zolotyh committed:
fix filter on mobile devices

zolotyh committed:
fix extrnal click for closing on touch devices

zolotyh committed:
remove focus

zolotyh committed:
add focus
